jetty-jspc-maven-plugin/pom.xml

```diff
@@ -11,6 +11,8 @@
   <name>Jetty :: Jetty JSPC Maven Plugin</name>
   <properties>
     <bundle-symbolic-name>${project.groupId}.jspc.plugin</bundle-symbolic-name>
+    <jacoco.skip>true</jacoco.skip>
+    <maven.test.skip.exec>true</maven.test.skip.exec>
```

Comment:
Why `maven.test.skip.exec` here?

Reply:
ah yeah good catch. I mixed some stuff when doing the same thing for 2 branches which are different sorry for that

Reply:
oh but wait the surefire plugin configuration.
see next commit :)